packages/example-app/package.json

```
@@ -10,17 +10,17 @@
     "test": "start-server-and-test \"yarn start\" http://localhost:3000 \"cypress run\""
   },
   "dependencies": {
-    "next": "^13.4.12",
+    "next": "^16.0.10",
     "next-query-params": "^5.1.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^19.2.3",
+    "react-dom": "^19.2.3",
     "use-query-params": "2.0.0"
```

While we're at it, would you mind incorporating this?

```suggestion
    "use-query-params": "^2.2.2"
```